README.md

```diff
@@ -1,2 +1,61 @@
 # Newsweek
 This project demonstrates the use of Twitter’s Bootstrap framework by building a mock-up of Newsweek site that actually uses it
+
+> The structure of the index page (index.html) is based on the Newsweek.com main page
+
+![screenshot](images/Screen_Shot_sm.png)
+![screenshot](images/Screen_Shot_md.png)
+![screenshot](images/Screen_Shot_lg.png)
+
+The structure of the index page is based on the Newsweek.com main page in small medium and large screens
+
+## Built With
+
+- HTML,
+- CSS,
+- Bootstrap
+- Google Fonts
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/ioanniskousis/Newsweek/595d3f237af2d17a6e638e8509277bbf2ee71c73/index.html)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+#### You should have installed git on your local machine and a text editor preferably VS Code.
+
+### Setup
+
+#### Clone the repository into your local machine.
+
+
+## 👤 Authors
+
+### Author1
```

Comment:
- [ ] There is only one author in this project.

Reply:
Corrected